roles/ceph-osd/tasks/docker/start_docker_osd.yml

```diff
@@ -1,4 +1,10 @@
 ---
+# For openstack VMs modify the mount point below depending on if the Openstack
+# VM deploy tool defaults to mounting ephemeral disks
+- name: umount ceph disk (if on openstack)
+  shell: "umount /mnt"
```

Comment:
can't you use the mount module (http://docs.ansible.com/ansible/mount_module.html) instead?